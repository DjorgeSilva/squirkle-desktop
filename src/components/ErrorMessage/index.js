// @flow

import React from 'react'
import styles from './styles.module.css'

type ErrorMessagePropTypes = {
  children: any
}

export default ({ children }: ErrorMessagePropTypes): React$Element<*> => {
  return <div className={styles.errorMessageContainer}>{children}</div>
}
