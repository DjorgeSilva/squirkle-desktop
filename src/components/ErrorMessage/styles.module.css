.errorMessageContainer {
	padding: 10px;
	text-align: center;
	font-weight: bold;
}
