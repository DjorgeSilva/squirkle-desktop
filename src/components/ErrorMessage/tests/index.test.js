// @flow

import React from 'react'
import { shallow } from 'enzyme'
import ErrorMessage from '../'

describe('ErrorMessage', (): void => {
  describe('When mounted', (): void => {
    let wrapper

    beforeAll((): void => {
      wrapper = shallow(<ErrorMessage>Something went wrong</ErrorMessage>)
    })

    it('renders correctly', (): void => {
      expect(wrapper).toMatchSnapshot()
    })
  })
})
