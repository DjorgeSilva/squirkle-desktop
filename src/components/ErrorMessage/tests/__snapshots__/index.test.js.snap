// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage When mounted renders correctly 1`] = `
<div
  className="errorMessageContainer"
>
  Something went wrong
</div>
`;
